cloud/types.go

```diff
@@ -220,15 +220,26 @@ type (
 
 	// DeploymentReviewRequest is the review_request object.
 	DeploymentReviewRequest struct {
-		Platform    string `json:"platform"`
-		Repository  string `json:"repository"`
-		CommitSHA   string `json:"commit_sha"`
-		Number      int    `json:"number"`
-		Title       string `json:"title"`
-		Description string `json:"description"`
-		URL         string `json:"url"`
+		Platform    string    `json:"platform"`
+		Repository  string    `json:"repository"`
+		CommitSHA   string    `json:"commit_sha"`
+		Number      int       `json:"number"`
+		Title       string    `json:"title"`
+		Description string    `json:"description"`
+		URL         string    `json:"url"`
+		Reviewers   Reviewers `json:"reviewers,omitempty"`
 	}
 
+	// Reviewer is the user's reviewer of a Pull/Merge Request.
+	Reviewer struct {
+		Login      string `json:"login"`
+		AvatarURL  string `json:"avatar_url,omitempty"`
+		GravatarID string `json:"gravatar_id,omitempty"`
```

Comment:
Do we really need this field? See https://developer.github.com/changes/2014-09-05-removing-gravatar-id/ and issues users have with this field https://github.com/orgs/community/discussions/38110

Reply:
Thanks for checking this! I will check if really needed.